packages/vue/src/components/organisms/SfContentPages/SfContentPages.html

```diff
@@ -0,0 +1,32 @@
+<div style="overflow: hidden">
+  <section
+    class="sf-content-pages"
+    :class="{'sf-content-pages--is-active': active}">
+    <div class="sf-content-pages__sidebar">
+      <h1 class="sf-content-pages__title">{{title}}</h1>
+      <SfList>
+        <SfList
+          v-for="page in pages"
+          :key="page"
+          class="sf-content-pages__list-item">
+          <!-- @slot -->
+          <slot
+            name="menu-item"
+            v-bind="{updatePage, page}">
```

Comment:
shouldn't we pass `active` here as well?

Reply:
right :) we should